pallets/api/src/messaging/transports/ismp.rs

```diff
@@ -166,15 +169,25 @@ impl<T: Config> IsmpModule for Handler<T> {
 impl<T: Config> IsmpModuleWeight for Pallet<T> {
 	// Static as not in use.
 	fn on_accept(&self, _request: &PostRequest) -> Weight {
-		DbWeightOf::<T>::get().reads_writes(2, 1)
+		DbWeightOf::<T>::get().reads_writes(1, 1)
```

Comment:
since it's not in use, maybe a `u64::MAX` would be more appropriate?

Reply:
Im worried this will the allow people to spam post requests which will be heavily overweighed. 

Although its "not in use" its still available to call and hence the weight should still be small.

Reply:
wouldn't they have to pay a large amount of funds to spam it?

Reply:
This weight for the moment isnt charged anywhere!